core/src/main/java/org/apache/struts2/components/UIBean.java

```diff
@@ -589,73 +586,72 @@ protected void mergeTemplate(Writer writer, Template template) throws Exception
     }
 
     public String getTemplateDir() {
-        String templateDir = null;
+        String result = null;
```

Comment:
Why did you rename this local variable?

Reply:
Recommended by SonarLint to keep local variable names different than the class fields names.